src/js/controllers/ctrl-create.js

```diff
@@ -497,6 +497,15 @@ ${msg.toLowerCase()}`,
 			}
 			// assumes questions is already a JSON string
 			questions = JSON.parse(questions)
+
+			//strip id from all imported questions and answers to avoid collisions
+			questions.forEach(question => {
+				question.answers.forEach(answer => {
```

Comment:
I'd like to add a safety check to make sure question.answers exists before this line